### PR TITLE
tests: rkt-monitor add new statistics

### PR DESCRIPTION
**Pull request by borqosky**
New statistics for load average and boot time measure. Godeps updated
as well

### CODE REVIEW

File: tests/rkt-monitor/main.go [hunk 1]
```diff
@@ -106,10 +107,13 @@ func runRktMonitor(cmd *cobra.Command, args []string) {
 		execCmd.Stderr = os.Stderr
 	}
 	err = execCmd.Start()
+
+	startTime := time.Now()
```
Comment (jellonek) [inline tests/rkt-monitor/main.go]: This line should be probably above above ;)

**Comment by jellonek** [comment]
Quite simple and nice. One minor flaw.

**Comment by jellonek** [comment]
LGTM

**Comment by pskrzyns** [comment]
+1

**Pull request closed without merging**